### PR TITLE
Quick entry order page,  checkout page design, cart page functions

### PR DESCRIPTION
**Pull request by kams2391**

### CODE REVIEW

File: app/Http/Controllers/CartController.php [hunk 1]
```diff
@@ -47,6 +50,9 @@ public function index(Request $request)
 			}
 		}
 		$cart=Cart::with('User','CartItem.Product.Media')->where('user_id', $user->id)->get();
+		/*echo "<pre>";
+		print_r($cart);
+		die;*/
```
Comment (upworksanjeev) [inline app/Http/Controllers/CartController.php:55]: @kams2391 remove this code

File: app/Http/Controllers/CartController.php [hunk 2]
```diff
@@ -34,7 +37,7 @@ public function index(Request $request)
 		}
 		$cartitems=CartItem::where('cart_id', $cart->id)->where('product_id', $request->product_id)->first();
 		if($cartitems){
-			$cartitems->update(['quantity' => $cartitems->quantity+1]);
+			$cartitems->update(['quantity' => $cartitems->quantity+1,'price' => $request->price,'discount' => $request->discount,'discount_price' => $request->discount_price]);
```
Comment (upworksanjeev) [inline app/Http/Controllers/CartController.php:40]: @kams2391 why we are using quantity+1 here?

File: app/Http/Controllers/CartController.php [hunk 3]
```diff
@@ -59,40 +65,155 @@ public function index(Request $request)
...
+							$cartitems->delete(); 
+						}	
+					    $cart->update(['total_items' => $cart_quantity]);						
+					}	
+				}
+		}
+		$cart=Cart::with('User','CartItem.Product.Media')->where('user_id', $user->id)->get();
 		
+
+		echo view('components.cart.product-list ', ['page' => 'cart','cart' => $cart]);
```
Comment (upworksanjeev) [inline app/Http/Controllers/CartController.php:111]: @kams2391 why we are using echo here?

File: app/Http/Controllers/CartController.php [hunk 4]
```diff
@@ -59,40 +65,155 @@ public function index(Request $request)
...
+     */
+    public function updateCartItemMarked(Request $request)
+    {
+		if($request->has('cart_item_id')){
+			$cartitems=CartItem::where('id', $request->cart_item_id)->first();
+			if($cartitems){
+				$cartitems->update(['marked_for' => $request->marked_for]);
+			}
+		}
+		echo 1;
```
Comment (upworksanjeev) [inline app/Http/Controllers/CartController.php:126]: @kams2391 why we are using echo here?

File: app/Http/Controllers/CartController.php [hunk 5]
```diff
@@ -59,40 +65,155 @@ public function index(Request $request)
...
+					foreach($cartitems as $v){
+						CartAttribute::where('cart_item_id', $v->id)->delete();
+						CartItem::where('id', $v->id)->delete(); 
+					}
+				}
+				$cart->delete(); 
+			}
+		}				
+		$cart=Cart::with('User','CartItem.Product.Media')->where('user_id', $user->id)->get();
+		echo view('components.cart.product-list ', ['page' =>  'cart','cart' => $cart]);
```
Comment (upworksanjeev) [inline app/Http/Controllers/CartController.php:149]: @kams2391 same here

File: app/Http/Controllers/CartController.php [hunk 6]
```diff
@@ -59,40 +65,155 @@ public function index(Request $request)
...
+	} 
+	
+	/**
+     * update cart item count in header
+     */
+    public function getCartCount(Request $request)
+    {
+		echo view('components.cart.cart-count');
```
Comment (upworksanjeev) [inline app/Http/Controllers/CartController.php:157]: @kams2391 why we are using echo to show view?

File: app/Http/Controllers/CartController.php (hunk 2 above)
Comment (kams2391) [reply app/Http/Controllers/CartController.php:40]: If product item already added in cart and user again add same product then it update that cart item quantity only instead to create new one.

File: app/Http/Controllers/CartController.php (hunk 3 above)
Comment (kams2391) [reply app/Http/Controllers/CartController.php:111]: I am using this function into ajax and in return update the cart view.

File: app/Http/Controllers/CartController.php (hunk 4 above)
Comment (kams2391) [reply app/Http/Controllers/CartController.php:126]: Using this function into ajax.  That is why using echo. For now 1 is static value.

File: app/Http/Controllers/CartController.php (hunk 2 above)
Comment (upworksanjeev) [reply app/Http/Controllers/CartController.php:40]: ok

File: resources/views/quick-entry.blade.php [hunk 7]
```diff
@@ -57,6 +57,72 @@ class="py-2.5 px-5 text-sm font-medium text-white focus:outline-none bg-[#008C99
...
+}
+function chooseProduct(sku,product_id){
+	$("#myTable").css("display","none");
+	$('#myInput').val(sku);
+	$('#selected_product_id').val(product_id);
+	$('#qty').val(1);
+}
+function addToCart(sku,product_id){
+	if($('#selected_product_id').val()==''){
+		alert("Enter Stock Code");
```
Comment (upworksanjeev) [inline resources/views/quick-entry.blade.php:93]: @kams2391 Don't use alert. Show proper error message under the field

File: app/Http/Controllers/CartController.php (hunk 4 above)
Comment (upworksanjeev) [reply app/Http/Controllers/CartController.php]: Use json response here

**Review by upworksanjeev — APPROVED** [review]

**Pull request merged**